src/main/java/cn/nukkit/player/Player.java

```diff
@@ -3595,7 +3595,7 @@ public void sendPosition(Vector3f pos, double yaw, double pitch, MovePlayerPacke
         MovePlayerPacket packet = new MovePlayerPacket();
         packet.setRuntimeEntityId(this.getRuntimeId());
         packet.setPosition(pos.add(0, getEyeHeight(), 0));
-        packet.setRotation(com.nukkitx.math.vector.Vector3f.from(yaw, pitch, yaw));
```

Comment:
The package can be removed from here

Reply:
not sure why that is even there.... IntelliJ did something weird there. Will remove.